packages/nvidia_nat_test/src/nat/test/plugin.py

```
@@ -722,3 +722,52 @@ def oauth2_client_credentials_fixture(oauth2_server_url: str, fail_missing: bool
         pytest.skip(reason=reason)
+
+
+@pytest.fixture(name="local_sandbox_url", scope="session")
+def local_sandbox_url_fixture(fail_missing: bool) -> str:
+    """Check if sandbox server is running before running tests."""
+    import requests
+    url = os.environ.get("NAT_CI_SANDBOX_URL", "http://127.0.0.1:6000")
+    try:
+        response = requests.get(url, timeout=5)
+        response.raise_for_status()
+        return url
```

Ruff (0.14.3)

Consider moving this statement to an `else` block

(TRY300)